BiasAndFairnessServers/src/utils/run_bias_and_fairness_check.py

```
@@ -0,0 +1,51 @@
+import pandas as pd
+import joblib
+from fairlearn.metrics import (
+    MetricFrame, 
+    selection_rate, 
+    true_positive_rate, 
+    true_negative_rate, 
+    demographic_parity_difference, 
+    equal_opportunity_difference, 
+    equalized_odds_difference
+)
+from sklearn.metrics import accuracy_score, classification_report
```

Ruff (0.11.9)

`sklearn.metrics.classification_report` imported but unused

Remove unused import: `sklearn.metrics.classification_report`

(F401)